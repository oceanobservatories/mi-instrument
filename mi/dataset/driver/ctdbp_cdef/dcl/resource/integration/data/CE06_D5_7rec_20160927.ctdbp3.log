2016/09/27 16:47:58.863 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/27 17:30:03.500 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/27 17:30:27.009  14.6181,  3.85568,    1.164, 727, 228, 83, 27 Sep 2016 17:30:21
2016/09/27 17:31:02.202 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/27 18:30:04.789 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/27 18:30:28.291  14.6318,  3.85918,    1.226, 637, 179, 83, 27 Sep 2016 18:30:23
2016/09/27 18:31:03.494 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/27 19:28:05.184 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/27 19:30:03.643 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/27 19:30:27.155  14.6834,  3.86364,    1.191, 589, 196, 78, 27 Sep 2016 19:30:22
2016/09/27 19:31:02.387 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/27 20:28:04.058 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/27 20:30:02.985 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/27 20:30:26.493  14.7828,  3.87224,    1.164, 585, 222, 82, 27 Sep 2016 20:30:21
2016/09/27 20:31:01.695 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/27 21:28:03.199 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/27 21:30:02.259 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/27 21:30:25.772  14.7956,  3.87753,    1.149, 1125, 316, 85, 27 Sep 2016 21:30:20
2016/09/27 21:31:00.968 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/27 22:28:05.129 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/27 22:30:04.517 [ctdbp3:DLOGP6]:Instrument Started [Power On]
2016/09/27 22:30:28.025  14.7422,  3.87221,    1.160, 1590, 385, 88, 27 Sep 2016 22:30:22
2016/09/27 22:31:03.221 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
2016/09/27 23:28:08.571 [ctdbp3:DLOGP6]:Logger started: Idle state, without initialize
2016/09/27 23:30:14.524 [ctdbp3:DLOGP6]:Instrument Started on second attempt [Power On]
2016/09/27 23:30:29.231  14.6939,  3.88162,    1.119, 1019, 423, 84, 27 Sep 2016 23:30:24
2016/09/27 23:31:04.434 [ctdbp3:DLOGP6]:Instrument Stopped [Power Off]
