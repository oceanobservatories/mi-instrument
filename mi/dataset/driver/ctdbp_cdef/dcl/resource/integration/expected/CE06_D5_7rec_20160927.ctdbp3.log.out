
 subsite  | node  | sensor       | bin        | method      | time                  | deployment | internal_timestamp | port_timestamp        | preferred_timestamp | quality_flag | conductivity | date_time_string | dcl_controller_timestamp | pressure | temp
----------+-------+--------------+------------+-------------+-----------------------+------------+--------------------+-----------------------+---------------------+--------------+--------------+------------------+--------------------------+----------+---------
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3683923200 | telemetered | 3683986227.0089998245 |          5 |         3683986221 | 3683986227.0089998245 |      port_timestamp |           ok |      3.85568 |             null |                     null |    1.164 | 14.6181
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3683923200 | telemetered | 3683989828.2909998894 |          5 |         3683989823 | 3683989828.2909998894 |      port_timestamp |           ok |      3.85918 |             null |                     null |    1.226 | 14.6318
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3683923200 | telemetered |  3683993427.154999733 |          5 |         3683993422 |  3683993427.154999733 |      port_timestamp |           ok |      3.86364 |             null |                     null |    1.191 | 14.6834
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3683923200 | telemetered | 3683997026.4930000305 |          5 |         3683997021 | 3683997026.4930000305 |      port_timestamp |           ok |      3.87224 |             null |                     null |    1.164 | 14.7828
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3683923200 | telemetered | 3684000625.7720003128 |          5 |         3684000620 | 3684000625.7720003128 |      port_timestamp |           ok |      3.87753 |             null |                     null |    1.149 | 14.7956
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3683923200 | telemetered | 3684004228.0250000954 |          5 |         3684004222 | 3684004228.0250000954 |      port_timestamp |           ok |      3.87221 |             null |                     null |     1.16 | 14.7422
 CE06ISSM | SBD17 | 06-CTDBPC000 | 3683923200 | telemetered | 3684007829.2309999466 |          5 |         3684007824 | 3684007829.2309999466 |      port_timestamp |           ok |      3.88162 |             null |                     null |    1.119 | 14.6939

(7 rows)
